src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/mapper/LimitedPartnershipIncorporationMapper.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.companieshouse.limitedpartnershipsapi.mapper;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.factory.Mappers;
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.limitedpartnershipsapi.model.dao.LimitedPartnershipIncorporationDao;
+import uk.gov.companieshouse.limitedpartnershipsapi.model.dto.LimitedPartnershipIncorporationDto;
+
+@Component
+@Mapper(componentModel = "spring")
+public interface LimitedPartnershipIncorporationMapper {
+
+    LimitedPartnershipIncorporationMapper INSTANCE = Mappers.getMapper(LimitedPartnershipIncorporationMapper.class);
```

Comment:
Will this need to map both ways?

Reply:
I don't think so. Well certainly not at the moment. When the POST end-point creates the Mongo resource initially it just creates a DAO directly (no mapping from DTO) and stores the JSON in Mongo.